Assets/Scripts/UI/Views/Popups/Pause/PauseMenu.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UI
+{
+    public class PauseMenu : PopupBase
+    {
+        [SerializeField] private Button _closeButton;
+        
+        private IUIRoot _uiRoot;
+        private PauseManager _pauseManager;
+        
+        private void OnEnable()
+        {
+            _closeButton.onClick.AddListener(Pause);
+        }
+
+        private void OnDisable()
+        {
+            _closeButton.onClick.RemoveListener(Pause);
+        }
+
+        private void Pause()
```

Comment:
Rename method to "OnCloseButtonClicked"

We use for handlers naming : "On<event name>"

Reply:
Done